Tests/FLXSerialExecutorSpec.m

```diff
@@ -53,20 +53,6 @@ @interface FLXSerialExecutor (Testing)
         [sut executeAndWait:block];
         [[theValue(didLaunchBlock) should] beTrue];
     });
-    
-    #ifndef DNS_BLOCK_ASSERTIONS
-    it(@"Should throw if no block passed to sync execution", ^{
-        [[theBlock(^{
-            [sut executeAndWait:nil];
-        }) should] raise];
-    });
-    
-    it(@"Should throw if no block passed to async execution", ^{
-        [[theBlock(^{
-            [sut execute:nil];
-        }) should] raise];;
-    });
-    #endif
```

Comment:
Is there a reason why we delete those tests? @degterev

Reply:
They didn't build, because method doesn't accept nil. Otherwise I can disable nullable errors for tests. @fuzza